rotkehlchen/api/v1/schemas.py

```diff
@@ -333,6 +333,12 @@ def validate_schema(
             )
 
 
+class EventsOnlineQuerySchema(AsyncQueryArgumentSchema):
+    name = fields.String(
+        validate=webargs.validate.OneOf(choices=('eth_withdrawals', 'block_productions')),
```

Comment:
Shouldn't this literals be in a list to avoid repeating them in the `resources.py` file too?